### ISSUE TITLE
Wrong HTTP GET parameter Curl::Easy.http_get

### ISSUE THREAD

**Issue by tempo** [taf2/curb#243]
TLDR: missing fix from commit `14e9b5312a15` in current gem on rubygems.org breaks `Curl::Easy.http_get`, please update gem

Hi there,

when I'm performing an HTTP GET request with the http_get method, strange HTTP
headers are sent to the server:

```http
GET /test HTTP/1.1
Host: localhost:5000
Accept: */*
Content-Length: -1
Content-Type: application/x-www-form-urlencoded
Expect: 100-continue
```

The Apache HTTP server responds with `HTTP/1.1 413 Request Entity Too Large`

This is my example request ruby code:
```ruby
require 'curb'

http_client = Curl::Easy.new
http_client.verbose = true
http_client.url = "http://localhost:5000/test"
http_client.http_get
```

And the corresponding verbose output:
```bash
$ ruby curl.rb
* About to connect() to localhost port 5000 (#0)
*   Trying ::1...
* connected
* Connected to localhost (::1) port 5000 (#0)
* 0x7f6b27a0d630 is at send pipe head!
* STATE: CONNECT => DO handle 0x7f6b27a2df40; (connection #0) 
> GET /test HTTP/1.1
Host: localhost:5000
Accept: */*
Content-Length: -1
Content-Type: application/x-www-form-urlencoded
Expect: 100-continue

* STATE: DO => DO_DONE handle 0x7f6b27a2df40; (connection #0) 
* STATE: DO_DONE => WAITPERFORM handle 0x7f6b27a2df40; (connection #0) 
* STATE: WAITPERFORM => PERFORM handle 0x7f6b27a2df40; (connection #0) 
* additional stuff not fine transfer.c:1037: 0 0
* additional stuff not fine transfer.c:1037: 0 0
* HTTP 1.1 or later with persistent connection, pipelining supported
< HTTP/1.1 413 Request Entity Too Large
< Date: Wed, 08 Apr 2015 11:18:18 GMT
< Server: Apache/2.4.10 (Debian)
< Connection: close
< Content-Type: text/html; charset=iso-8859-1
< 
* nread <= 0, server closed connection, bailing
* STATE: PERFORM => DONE handle 0x7f6b27a2df40; (connection #0) 
* Closing connection #0
* Expire cleared
```

This is on debian stable with `libcurl3 7.26.0-1+wheezy12` and `curb 0.8.7` with ruby 2.1.5

There are no problems with Curl::Easy.perform and Curl::Easy.http

It looks like a missing `break` in the file `ext/curb_easy.c` file which is fixed in commit `14e9b5312a15de7a243c29667b53c2157d8fef61` from 2015-03-25:

```git
git diff 14e9b5312a15de7a243c29667b53c2157d8fef61^ 14e9b5312a15de7a243c29667b53c2157d8fef61
diff --git a/ext/curb_easy.c b/ext/curb_easy.c
index 7c13d01..7db1e3a 100644
--- a/ext/curb_easy.c
+++ b/ext/curb_easy.c
@@ -3125,6 +3125,7 @@ static VALUE ruby_curl_easy_set_opt(VALUE self, VALUE opt, VALUE val) {
   case CURLOPT_NOPROGRESS:
   case CURLOPT_NOSIGNAL:
   case CURLOPT_HTTPGET:
+    break;
   case CURLOPT_POST: {
     curl_easy_setopt(rbce->curl, CURLOPT_POST, rb_type(val) == T_TRUE);
   } break;

$
```

The curb gem from rubygems.org is uploaded 3 days before this commit on 2015-03-22, therefore the `break` is not included.

By the way: `libcurl 7.38.0-4` on debian testing generates the following HTTP header for the same request:
```bash
GET /test HTTP/1.1
Host: localhost:5000
Accept: */*
Content-Type: application/x-www-form-urlencoded
Expect: 100-continue
```

**Comment by taf2** [comment]
0.8.8 is pushed
